SECURITY.md

```diff
@@ -18,6 +18,14 @@ As Git for Windows bundles more than just Git (such as Bash, OpenSSL, OpenSSH, G
 
 Every Git for Windows version is tagged using a name that starts with the Git version on which it is based, with the suffix `.windows.<patchlevel>` appended. For example, Git for Windows v2.17.1' source code is tagged as [`v2.17.1.windows.1`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.1) (the patch level is always at least 1, given that Git for Windows always has patches on top of Git). Likewise, Git for Windows v2.17.1(2)' source code is tagged as [`v2.17.1.windows.2`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.2).
 
+## Release Candidates (rc) and Snapshot versions ('nightlies')
+
+Git for Windows also releases versions that reflect the [upstream release candidates](https://tinyurl.com/gitCal). These contain the `-rc<n>` suffix to the expected regular git version, and before the 'windows' suffix. These releases are independent of upstream but are tied together by convention. It should be noted that these rc versions currently sort after their formal release, so appear to be newer to the updater software.
+
+[All releases](https://github.com/git-for-windows/git/releases/) are listed via a link at the footer of the [Git for Windows](https://gitforwindows.org/) home page.
```

Comment:
Maybe you could mention here that Release Candidates as well as MinGit backports are marked as "prereleases", so that the latest version will always be a stable, full Git for Windows?

Oh, and it would probably be a good idea to split out the section talking about the snapshot versions. They are built from `master`, are hosted somewhere completely different than the Release Candidates and the full Git for Windows releases, and at some stage in the future I might have to start garbage-collecting old snapshots, while I will never need to do that with the RCs/full versions. So they are quite different in nature, and hence would merit being handled in different sections.